### DISCUSSION TITLE
Can't login via Desktop/Mobile Client - but Webclient works ...

### DISCUSSION THREAD

**Discussion by PeterIttner** [dani-garcia/vaultwarden#3025] [Q&A]
### Subject of the issue

Can't login with official bitwarden Desktop and Mobile Clients to self-hosted vaultwarden anymore.
Login via web-interface works (therefore I know my credentials are correct).

### Deployment environment

* vaultwarden version: 1.22.2

* Install method: Docker running in Synology NAS
* Clients used: Desktop Client 2022.12.0
* Reverse proxy: Vaultwarden is running in local network with a reverse proxy that handles https.
* Database: SQLite with  `/data` directory is mounted to local volume via docker

### Steps to reproduce

1. Start Bitwarden Desktop Client
2. Configure Server URL in settings (including https://...)
3. Login with username (email) and masterpassword

### Expected behaviour

Should be logged in, like in web-client

### Actual behaviour

Error: Username or Password incorrect

### Troubleshooting data

Docker log: 
```
[2022-12-21 11:01:49.194][vaultwarden::api::identity][ERROR] Username or password is incorrect. Try again. IP: XXXXX. Username: YYYYY.
```

**Comment by BlackDex** [comment]
Please search before posting.
There are a lot of similar posts already.

Update to the latest version!
